ext/Server/Bot.lua

```diff
@@ -32,6 +32,8 @@ function Bot:__init(player)
 	self._wayWaitTimer = 0;
 	self._wayWaitYawTimer = 0;
 	self._obstaceSequenceTimer = 0;
+	FirstShotRandomness = math.random (0, 2) / 10;
```

Comment:
Lets use MathUtils:GetRandom(from, to) here
This looks nicer and returns directly in float
```suggestion
	FirstShotRandomness = MathUtils:GetRandom(0, 0.2);
```